### PR TITLE
(feat)add translations to reports module

### PR DESCRIPTION
**Pull request by jona42-ui**
# Objective
Add translations to report module

# Screens
Screenshot from 2023-12-05 23-53-35
Screenshot from 2023-12-05 23-53-57
Screenshot from 2023-12-05 23-54-18
Screenshot from 2023-12-05 23-54-37
Screenshot from 2023-12-05 23-55-04

Screenshot from 2023-12-06 12-35-22

### CODE REVIEW

**Comment by jona42-ui** [comment]
cc @mozzy11

**Comment by mozzy11** [comment]
Thanks @jona42-ui .
can you attach the screenshots for  translated pages for both languages?
Also, you don't need to include  `defaultMessage="Patient Status Report"`  as the [translation logic ](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/2.8/frontend/src/App.js#L96-L98)already includes a default message

**Comment by jona42-ui** [comment]
> Thanks @jona42-ui . can you attach the screenshots for translated pages for both languages? Also, you don't need to include a default message as the [translation logic ](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/2.8/frontend/src/App.js#L96-L98)already includes a default message
> 
> ie instead of `<FormattedMessage id="sidenav.label.statusreport" defaultMessage="Patient Status Report"/>` , just do `<FormattedMessage id="sidenav.label.statusreport"/>`

thanks @mozzy11 
can you now look at the screen cast for both pages?
Sure, let me revert the `default` thing

**Comment by jona42-ui** [comment]
Screenshot from 2023-12-01 11-03-12
@mozzy11  what may be the cause of this, it consistently happens when I access the instance in the browser.

[backend_logs.txt](https://github.com/I-TECH-UW/OpenELIS-Global-2/files/13522664/backend_logs.txt)

**Comment by jona42-ui** [comment]
This has persisted am not sure how to get over this.
am not sure if its my machine with issues

**Comment by jona42-ui** [comment]
the error still persists, what i did was;
`sudo docker-compose -f dev.docker-compose.yml down
 `
then;
`sudo docker volume rm openelis-global-2_db-data
`
then;

`sudo docker-compose -f dev.docker-compose.yml up -d
`
just I got proxy issues and remapped these ports
portsm

**Comment by mozzy11** [comment]
can we jump on a call

**Comment by jona42-ui** [comment]
sure
sorry i was off abit

**Comment by jona42-ui** [comment]
can we do on meet?
https://meet.google.com/fky-czvm-gia

**Comment by mozzy11** [comment]
can you ping caesy on this [thread](https://talk.openelis-global.org/t/quick-note-on-this-community-page/125/2)

**Comment by mozzy11** [comment]
can you also fix your merge conflicts ??

**Comment by mozzy11** [comment]
are you there ??

**Comment by mozzy11** [comment]
were you able to run the setup ??

**Comment by jona42-ui** [comment]
Yes i did when I sent the  other PR but now its the same case

**Comment by jona42-ui** [comment]
its now failing again

**Comment by jona42-ui** [comment]
am sorry moses lyca betrayed me

**Comment by jona42-ui** [comment]
yes let me log on

**Comment by jona42-ui** [comment]
cool on it. thanks

*Sent with Right Inbox <https://www.rightinbox.com/?utm_source=signature>*

**Comment by jona42-ui** [comment]
its up now. all oaky after spinning up fresh containers

**Comment by jona42-ui** [comment]
Sorry i need to make more clean ups here

**Comment by mozzy11** [comment]
@jona42-ui can you attach screen shots of both the pages you've added translations ??

**Comment by jona42-ui** [comment]
ooh! okay though basically it was menu items
I was wondering how to  translate the forms/reports from the dynamically constructed URLs

**Comment by mozzy11** [comment]
are you able to run openelis succesully ??

**Comment by jona42-ui** [comment]
yes its running successfully

**Comment by mozzy11** [comment]
Alright , can you attach the screenshots of the translated pages ??

**Comment by mozzy11** [comment]
>  was wondering how to translate the forms/reports from the dynamically constructed URLs

i ddnt get you comment above

**Comment by jona42-ui** [comment]
So, just to be sure am getting this well,, what is attached now above in the description  is not what you expect?

**Comment by mozzy11** [comment]
are those the updated screenshots that reflect your current changes ??

**Comment by mozzy11** [comment]
are you done with any cleanups you wanted to do ??
ensure you update the screenshots that reflect your current changes

**Comment by jona42-ui** [comment]
alright @mozzy11  just winding up, will make sure all is up to date

**Comment by jona42-ui** [comment]
Thanks

**Comment by jona42-ui** [comment]
Updated!
@mozzy11

**Comment by jona42-ui** [comment]
As a follow up in another PR is translating this;
Screenshot from 2023-12-05 23-58-42

though I will need guidance on how!

File: frontend/src/components/Reports/Routine.js [hunk 1]
```diff
@@ -10,137 +11,138 @@ export const RoutineReportsMenu = {
...
-      title: "Activity Report",
+      title: <FormattedMessage id="sideNav.title.activityreport"/>,
       icon: IbmWatsonNaturalLanguageUnderstanding,
       SideNavMenuItem: [
         {
           link:
             config.serverBaseUrl +
             "/Report?type=indicator&report=activityReportByTest",
-          label: "By Test Type",
+          label: <FormattedMessage id="sideNav.label.bytesttype"/>,
```
Comment (mozzy11) [inline frontend/src/components/Reports/Routine.js:69]: did you just forget the english translation for this ??

**Pull request merged**